docs/install-and-run.md

```diff
@@ -254,7 +254,7 @@ steps in your setup:
     #import <EarlGrey/EarlGrey.h>
    ```
 
-2. Download [`EarlGrey.swift`](https://github.com/google/EarlGrey/tree/master/gem/lib/earlgrey/files/EarlGrey.swift) and add it to your test bundle. The file contains
+2. Download [`EarlGrey.swift`](https://github.com/google/EarlGrey/tree/master/gem/lib/earlgrey/files/Swift-3.0/EarlGrey.swift) and add it to your test bundle. The file contains
```

Comment:
@bootstraponline 

we should be clear that this is flow is for manually including EarlGrey.swift outside of pods flow. Pods provides an automated way of doing that. Also, could we link the user to a directory where they can choose which version to get? I'd image when swift 3.1 or 4.0 comes out, we'll have to constantly update this...We could create a latest folder and symlink it to the latest and have our code and the default swift version in earlgrey gem reference that always?